Formula/g/gyb.rb

```diff
@@ -17,16 +18,18 @@ class Gyb < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "4bf46d6ec62b1f3cfbaf289b797c831811d1bb13691811d62ddee6584cea79bf"
   end
 
-  depends_on "pyinstaller" => :build
   depends_on "python-certifi" => :build
-  depends_on "python-pyparsing" => :build
-  depends_on "python-setuptools" => :build
+  depends_on "python-cryptography" => :build
```

Comment:
Shouldn't this be a runtime dep?

Reply:
I think `pyinstaller` bundles everything into self-contained app.

This also means it has copies of `openssl@3` libraries that will not be updated unless we rebuild bottles. Using an extractor, the dylibs bundled include:
```
libcrypto.3.dylib
liblzma.5.dylib
libmpdec.4.dylib
libsqlite3.0.dylib
libssl.3.dylib
```

Reply:
Meant to circle back here, but yeah – `pyinstaller` bundles most everything it can, including the cryptography package. I hadn't realized that it would also include all the dylibs.

Do you think, for that reason, if we removed `depends_on "openssl@3"` and add a `rebuild`, that it should still pass CI?

Reply:
.. and I missed that you already added a `rebuild`, but left the `openssl@3` dependency in, so I guess that means it is still needed as a runtime dep?

Reply:
The `rebuild` is just because we didn't rev bump this. BrewTestBot adds it when pushing the bottles to differentiate from the original ones.

Reply:
I may try switching to Nix's installation https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/tools/backup/gyb/default.nix#L30-L32

And maybe blacklist `pyinstaller`. We did blacklist `pyoxidizer` (https://github.com/Homebrew/brew/pull/12493) but that sounds even worse as it also used non-Homebrew `rust`/`python`.

Reply:
Blacklisting `pyinstaller` makes sense.